backend/modules/simulation/agents/persona_agent.py

```
@@ -37,6 +37,63 @@
 debug_log = logging.getLogger(__name__).debug
 logger = logging.getLogger(__name__)
 
+# ─── Big Five behavioral descriptors ─────────────────────────────────────────
+# Maps score ranges to plain-language behavioral descriptions used in system prompts.
+# Scores follow the Big Five model: openness, conscientiousness, extraversion,
+# agreeableness, neuroticism — each on a 1–10 scale.
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)